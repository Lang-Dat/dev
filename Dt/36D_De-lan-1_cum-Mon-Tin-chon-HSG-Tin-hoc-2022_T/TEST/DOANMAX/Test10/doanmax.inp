1000
-7338 -7163 54 3850 1817 -8539 5428 2800 5701 -7540 -9037 -8707 -4770 4600 -5504 -361 8514 -3702 -7939 4740 8369 3832 7280 3114 -6276 8645 6159 2495 1118 7089 -889 -1409 8139 4032 2031 -2461 2649 1733 -6793 3445 -3837 -6730 -8130 -796 -6674 3744 918 -2021 2872 -4272 7502 -9577 2740 442 -5014 4290 -3754 9405 -5859 9330 -6585 -7012 3351 -9027 -6201 -3595 1695 4953 8718 3495 9686 7338 3604 5108 1690 7767 -566 -3452 -1849 6588 3031 -5031 6360 -4959 3301 -6580 4473 -5351 5813 3015 6893 -9177 4607 -9151 1003 -9836 -5770 -1957 -4471 5154 8123 -4769 9456 -3057 -9617 -4917 8380 -9063 2460 -3827 -7405 3679 -1816 4589 -4344 2932 3829 -2856 8102 2902 -2762 -7551 -8962 1424 3683 -7584 -6667 6882 -295 2393 -722 -7999 3381 3050 3978 8450 -8481 -7762 -6050 3845 -8212 2125 2028 5833 -3586 8084 7324 -515 -3954 -2839 -1981 5901 3117 9791 2753 10 3753 2104 -3840 9264 6795 3240 4841 8739 -1936 -5646 571 -2480 6009 -7737 8034 1661 9247 5589 4692 995 -5214 5516 -4366 -8963 1140 9668 -847 -400 2961 -7226 2863 -6952 3727 -886 -9617 1912 -2704 9686 7244 -8455 9110 -6938 9117 5573 2527 8505 3789 5499 8073 798 -7619 -2451 7681 3309 3349 6229 -2930 8007 923 -9612 5005 -6325 7606 6431 -7681 91 2451 -1495 1743 -8278 6927 -6373 -5363 6929 4962 -8314 -4479 -8963 298 805 1836 75 9350 6521 -1680 -123 2959 -4964 -3018 8239 -8632 1000 8382 1793 -3575 -6274 -8192 9585 -3130 -8956 3747 6279 -9597 6782 9972 1137 -5386 6233 8192 9376 2615 -9254 57 696 6659 -3928 3183 -736 -3018 -5930 -1026 9517 444 3677 84 -8727 -9417 502 -2648 819 1784 2518 6030 2459 -6672 -4702 -6899 -3778 -8785 -3690 -360 3232 -7367 5599 7691 9867 773 -5820 -5464 -7215 27 9627 -7754 5889 -994 -8880 -5582 8030 -2745 5489 223 3925 5526 -5340 -2952 -3877 -3321 5451 5537 8789 7626 -7097 2311 5629 -232 157 -5990 5544 2401 9988 -3575 -4472 4311 -6538 3713 -4625 1809 -375 2809 3443 5184 9030 -1690 8296 -824 4805 -4223 -8109 6214 2375 -119 -5587 8396 -5628 -8003 -7631 -6185 -2899 -1441 -4986 -253 8905 1400 -8045 -3882 4760 945 -6868 7868 -747 438 3031 186 2275 -8178 7464 -1400 7052 -6621 3615 9977 5223 5433 -3443 4966 6036 1158 -1872 8835 -8464 -480 805 2384 4756 -2837 -8106 -4163 -1920 -65 9603 8705 9000 6278 2746 7650 5076 1728 -4276 8120 3446 2385 -519 -1288 -6404 -7314 6273 -5276 -2741 -3134 -6528 792 1194 2068 5211 -2990 1703 9518 -9709 364 6971 -7632 -7778 -1678 -7077 9431 7209 5559 -8957 -7773 -3190 3470 -5894 9089 -7655 -9714 9172 2748 2740 6616 9851 -6874 -1261 -8684 4757 8590 4832 4527 9713 -6930 1588 8604 -9529 -6288 -4792 3732 -1586 -2059 -8671 7916 6075 -5938 6762 -3947 -7018 2435 -1798 9563 4181 -7533 8581 8296 838 6522 6301 9824 -9948 -3734 -7311 5248 7962 -2042 -5849 3893 9872 -8075 3672 -3230 9458 -2571 -3145 4773 1803 8483 -4517 7648 -2179 -8002 7499 -356 -8180 4912 -4560 8221 -9208 -7473 -213 337 -6273 -6678 6207 -3167 1450 -8797 -4668 285 -1645 8232 5192 -9646 -5217 -6197 -2154 -9674 7673 1411 -9679 -3445 -8944 6888 -3324 7433 820 -8588 243 1960 -8617 9789 -9550 17 3750 -4448 -4176 -7910 2533 4839 7639 -4013 8715 -5126 -5515 9959 4646 1675 1575 -8105 9519 2336 4252 -2694 -4235 6129 -5782 6279 7839 -8222 4867 -8544 -4645 2143 -1530 8833 -3629 -7348 4009 -488 -4799 -68 9227 7555 -8780 6362 3757 -4872 667 9806 8401 -5358 -9672 -4672 9536 -6271 -2813 4514 5119 -2159 7424 5822 -584 2473 -574 -7767 -4601 4660 211 -2853 -3090 5581 -9435 2756 8095 -813 -7618 7273 -8560 2999 -2268 346 -7764 7215 3907 -630 9428 3182 7630 2581 -6630 6133 -2094 -5200 -321 -5719 -356 -9350 -133 -3307 -225 8072 -5472 -9882 -8494 6738 -217 3753 5922 5402 -9 3988 4461 -7924 -8232 -2776 2753 -6179 -2372 -5353 -3110 -7769 6601 -3772 3254 6192 -2320 4945 2844 -6598 3449 -635 -1977 1728 -7413 -4087 -7131 2437 -8479 -3461 -5725 -3621 6964 -1302 4744 3431 -5352 7814 7072 -2627 -5119 -8285 -5986 1593 -7211 7385 1894 -4461 2020 -2805 3557 349 6290 -1709 7648 2112 -2581 -7756 8279 4317 -2003 -9313 -2275 6855 9579 6474 -5885 -2951 3693 9298 -376 -2972 7779 -1410 -4632 2602 -5014 3757 1114 -5455 -3306 2078 -2920 9141 -6475 1155 9147 6124 -5616 -7067 -3293 9955 -6577 -2269 -8958 3386 -2105 5497 1608 -3387 877 -6722 442 -5903 -5626 -2512 -5303 -9072 3277 6732 -7963 517 -5976 9587 -8856 -3088 -4811 -9505 -9288 -725 3232 3361 1355 -8738 -9067 265 5379 8392 -7104 -8375 -7524 -9368 9774 3597 1356 -6318 -6112 -4396 -3029 -3478 8876 -8120 3154 3098 7522 5914 -232 -6367 -22 -5113 6160 -4818 -4488 -7174 -7357 -8458 688 3265 9539 4043 -8383 2541 -3612 5683 -396 -6910 -6768 -1441 -7464 8289 -317 9461 -3491 8185 5219 -2437 -7993 1610 -9448 -8400 -2972 5539 9831 3712 592 -5284 4877 -4079 822 -8562 2839 -1216 -1071 -5754 -3224 -6660 7181 5527 -9024 554 -3575 -3744 1640 5745 346 782 -4346 -6143 7093 -432 7564 -2183 5458 -4178 923 2200 9430 -2763 4631 -6320 -4253 1434 9683 -1644 9301 -4944 1452 8385 -4837 -3758 9111 5935 -8030 -8004 2377 8846 -3260 3273 -601 -1475 -4310 -7845 8430 -8320 -4905 9161 -8751 6252 -1850 5265 7308 -2219 -2840 -2488 -1409 7897 -2270 8289 -573 -8182 -7967 1244 7034 2903 4933 2969 3026 -2052 -8919 -1272 5301 -243 5595 3762 -8766 -1281 1494 -9901 6343 2410 8578 -5155 1324 7281 -2103 7618 -5855 8159 -2249 -7944 5839 9604 -5629 -4870 1458 6347 2453 -253 9205 -9069 6896 -1416 1992 6877 -8374 9889 4305 -4533 -4300 6484 -6407 -3918 2887 -9970 8451 203 7584 -706 -4917 3324 -4327 -8971 -2477 -6574 -6661 -5773 4819 -6784 5428 6101 -5094 -6542 -8240 9794 450 4041 802 -4769 3903 
