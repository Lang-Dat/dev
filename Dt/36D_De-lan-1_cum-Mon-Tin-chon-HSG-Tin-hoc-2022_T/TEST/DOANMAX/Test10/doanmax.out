6
-9027 -6201 -3595 1695 4953 8718 